components/_patterns/00-base/layouts/_default.scss

```diff
@@ -53,18 +53,18 @@ $main-width: 100 - ($sidebar-width + #{$gutter});
 //   &--with-sidebar {
 //     @include large {
 //       display: grid;
-//       grid-gap: 2%;
+//       grid-gap: $gutter;
 //       grid-template-areas: "content sidebar";
-//       grid-template-columns: $main-width $sidebar-width;
+//       grid-template-columns: $main-width #{$sidebar-width};
 //     }
 //   }
 // }
-//
+
 // .main-sidebar {
 //   grid-area: sidebar;
-//   margin-bottom: $space-double;
+//   margin-bottom: $gutter;
```

Comment:
I like what's been done here, but I personally prefer to use space for bottom margins between things. I also like it to be space-double instead of just space visually. If you prefer to have these be the same, let's make a gutter-double that uses space-double. We could also just use space for all of these instead of making our own vars just for this file. What do you think?

Reply:
I'm fine with this the way it is. This is just an example after-all. :)

Reply:
I think the more vs less vertical space is probably a personal preference thing, so I'm not going to push back.

Let's :shipit: !